mobile/lib/services/filedata/filedata_service.dart

```diff
@@ -29,6 +29,12 @@ class FileDataService {
     _prefs = prefs;
   }
 
+  void appendPreview(int id) {
```

Comment:
Can you document what this method does and why do we need it? Unable to infer quickly while looking that the code

Reply:
I have documented, let me know if it seems fine?

Reply:
Understood. Don't we have objectID and size information avoid at this point? If yes, why not just append that?

Reply:
Ok got it, we do have that info as it is just uploaded.